Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/Resources.mustache

```diff
@@ -598,49 +597,82 @@ namespace {{ResourceClassesNamespace}}
             {{#Usi}}
             {{#Property}}
             // Property
+            {{#IsNullable}}
             if ((this as {{NamespacePrefix}}I{{ClassName}}).{{PropertyName}} == null
                 || !(this as {{NamespacePrefix}}I{{ClassName}}).{{PropertyName}}.Equals(compareTo.{{PropertyName}}))
                 return false;
+            {{/IsNullable}}
+            {{^IsNullable}}
+            if ((this as {{NamespacePrefix}}I{{ClassName}}).{{PropertyName}}.Equals(compareTo.{{PropertyName}}))
```

Comment:
I think I understand what the template is doing. The question is, why make this change now? Is this solving some known bug?

Reply:
the compiler does not respect the pragma statement anymore, so i needed to fix code gen so the application would compile